three_points_calibration:
  ros__parameters:
    PHOTO_POSE: [32.843, -9.295, 10.193, 0.00, -90.00, 32.690]
    camera_intrinsics: [1362.3206884245012, 0.0, 937.5666174434277, 0.0, 1360.1275923037902, 552.6777467209566, 0.0 ,0.0, 1.0]
    camera_distortion: [0.18133544672027332, -0.5755924836142977, 0.0024102913275606277, -0.0007547425526466893, 0.5218626472027835]
    tool2cam_quaternion: [0.007049283965255718, -0.0037376991168765208, -0.015327845036299319, 0.9998506860363862] #x,y,z,w
    tool2cam_trans: [-0.02760361078502668, -0.03365709503264675, 0.03654466986285128]
    base2tool_quaternion: [-180.0, 0.0, 90.0] #angle
    base2tool_trans: [-0.229737, 0.355894, 0.294339]
    input_pin: 6